libs/testing/src/jobs.service.ts

```diff
@@ -11,26 +11,20 @@ export class JobsService {
 
   public standardQueue: Queue;
   public workflowQueue: Queue;
+  public subscriberProcessQueue: Queue;
```

Comment:
I think that if you want this structure we should go for `subscriberProcessingQueue` naming.